test/contracts/elrondei/elrondei.c

```diff
@@ -0,0 +1,88 @@
+#include "../elrond/context.h"
+
```

Comment:
Add extra test to call `executeOnSameContext` to another function in another contract, function which takes the call value and calls `transferValue` / 2 to some hard-coded address.

Reply:
Will come as a next PR, after `executeOnSameContext` is fixed.